src/python3_pip_skeleton/__main__.py

```diff
@@ -127,8 +127,10 @@ def replace_text(text: str) -> str:
         return text
 
     def replace_in_file(file_path: Path, text_from: str, text_to: str):
-        file_contents = file_path.read_text()
-        file_path.write_text(file_contents.replace(text_from, text_to))
+        file_contents = file_path.read_text(errors="ignore", encoding="utf-8")
```

Comment:
Does this work if you just force the encoding to utf8 without setting `errors="ignore"`?

Reply:
Seems to, I've removed it.